upstream-sha256: 175aeaaa3eccb4b58595e43fcf03df8f5e7078ffe79e4e0009cec0ae478f03ed
actual-sha256: 6b7e503f2c32fd11082a1bc321c9f77739ad21c626e3172115b2991b737df21d
--- upstream.expected
+++ tolk.actual
@@ -4,5 +4,5 @@
   float4 val1 = (*((float4*)((data1_16+12))));
   float4 val2 = (*((float4*)((data1_16+4))));
   float4 val3 = (*((float4*)((data1_16+8))));
-  *(data0_1+0) = -__builtin_sqrtf((val0[0]+val0[1]+val0[2]+val0[3]+val2[0]+val2[1]+val2[2]+val2[3]+val3[0]+val3[1]+val3[2]+val3[3]+val1[0]+val1[1]+val1[2]+val1[3]));
+  *(data0_1+0) = -__builtin_sqrtf((0.0f+val0[0]+val0[1]+val0[2]+val0[3]+val2[0]+val2[1]+val2[2]+val2[3]+val3[0]+val3[1]+val3[2]+val3[3]+val1[0]+val1[1]+val1[2]+val1[3]));
 }
